temperature/router.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import APIRouter, Depends
+from fastapi_pagination import add_pagination
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import Session
+
+from temperature import schemas, crud
+import dependecies
+
+router = APIRouter()
+add_pagination(router)
+
+
+@router.get("/", response_model=list[schemas.Temperature])
+async def get_temperatures(db: AsyncSession = Depends(dependecies.get_db)):
+    return await crud.get_temperatures(db=db)
+
+
+@router.post("/update/")
```

Comment:
Where do you specify /temperature prefix for endpoints?

Reply:
main.py 21-25